### ISSUE TITLE
Open file dialog that allows selecting both files and folders

### ISSUE THREAD

**Issue by toadzky** [tauri-apps/plugins-workspace#2137]
I was looking at rebuilding a macOS app I maintain using Tauri, but the app lets users select a mix of files and folders. Tauri only lets me open a file dialog that can select files _or_ folders, but not both.

**Comment by Zhou-Haowei** [comment]
Hi! I was looking for the solution of this feature too. Any updates ?

**Comment by chnykn** [comment]
me too

**Comment by FabianLars** [comment]
To explain what the `status: upstream` label means: The rust crate we use for dialogs (https://github.com/PolyMeilex/rfd) does not support this yet and we didn't have the resources to create a PR ourselves so far.

**Comment by BarraR3port** [comment]
Any updates about this?

**Comment by FabianLars** [comment]
Nope, no traction whatsoever in the issue linked (https://github.com/PolyMeilex/rfd/issues/131)

**Comment by thewh1teagle** [comment]
But can we open files OR folders in one click? (not both)

**Comment by FabianLars** [comment]
No, you have to choose before opening the dialog and the dialog itself will filter the type upfront.

**Comment by imjeehoo** [comment]
Upstream implemented the feature: https://github.com/PolyMeilex/rfd/pull/290
